### PR TITLE
SpdxDeclaredLicenseMapping: Add an entry for `PSFL`

### PR DESCRIPTION
**Pull request by fviernau**
Signed-off-by: Frank Viernau <frank.viernau@here.com>

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1655)

### CODE REVIEW

File: spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt [hunk 1]
```diff
@@ -259,6 +259,7 @@ object SpdxDeclaredLicenseMapping {
         "Public domain (CC0-1.0)" to CC0_1_0.toExpression(),
         "PublicDomain" to licenseRef("public-domain-disclaimer", "scancode"),
         "Python Software Foundation" to PYTHON_2_0.toExpression(),
+        "PSFL" to PYTHON_2_0.toExpression(),
```
Comment (mnonnenmacher) [inline spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt]: This entry should be above "Perl Artistic v2" to match the sorting in this file.

Comment (fviernau) [reply spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt]: done.

Comment (fviernau) [reply spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt]: lol, indeed it was `above`.

**Comment by mnonnenmacher** [comment]
Merged, ignoring the unrelated error on AppVeyor.

**Pull request merged**